liminal/logging/logging_setup.py

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+from liminal.core import environment
+LIMINAL = 'liminal'
+LOGS_DIR = 'logs'
+
+def logging_initialization():
+    # TBD - rotating file handler
```

Comment:
I see these are still TBD
Currently I'm not seeing the log lines in sysout or in logs file
I think for now let's just focus on sysout and not log to file

Reply:
<img width="1427" alt="Screen Shot 2020-12-28 at 23 16 03" src="https://user-images.githubusercontent.com/1331783/103246404-98423d80-4963-11eb-8adc-bc658e7c1e60.png">
I see the logs both in the file & sysout, can u pls specify ?

Reply:
The TBD are for future reference/work, I can remove them if you prefer.